### ISSUE TITLE
Navigation can be improved in: "Users > My Profile" and "Users > Personal Settings" 

### ISSUE THREAD

**Issue by AtrumGeost** [Automattic/wp-calypso#51343]
**What**

1. Click on "Users > My Profile" or  "Users > Personal Settings" 
2. Notice that you are taken form the WordPress.com Dashboard (Nav Unification) to the https://wordpress.com/me navigation

On Atomic sites, this can also create confusion since editing "My Profile" won't work the same as on /wp-admin/profile.php

**How**
It would be great if we can:

- Have the same navigation experience (e.g. load the "My Profile" and "Personal Settings" without changing navigation), or…
- Have a way for the user to know where they landed and how to get back

**Why**
For new users, especially if they are not tech-savvy this can be confusing. And the experience is not consistent.

**Additional context**

**Comment by arunsathiya** [comment]
Yes to bridging this gap. Here's an example of a profile having "Disable the visual editor when writing" setting enabled on the wp-admin profile settings page, and it was not evident until we directly visited that page: p1656037179501359/1656035307.426459-slack-C03TY6J1A 

Showing wp-admin profile (domain.com/wp-admin/profile.php) settings on Calypso profile settings (https://wordpress.com/me) would be nice.
